tools/Datadog.Core.Tools/Datadog.Core.Tools.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net452;net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net45;net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
See [this comment](https://github.com/DataDog/dd-trace-dotnet/pull/624/files#r375458816).